x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/extractor/CompositeKeyExtractor.java

```diff
@@ -54,7 +55,7 @@ public CompositeKeyExtractor(String key, Property property, ZoneId zoneId, DataT
     CompositeKeyExtractor(StreamInput in) throws IOException {
         key = in.readString();
         property = in.readEnum(Property.class);
-        if (in.getTransportVersion().onOrAfter(INTRODUCING_UNSIGNED_LONG_TRANSPORT)) {
+        if (in.getTransportVersion().onOrAfter(TransportVersion.fromId(INTRODUCING_UNSIGNED_LONG.id))) {
```

Comment:
This mapping is not a good idea. It should be as it was - referencing a specific `TransportVersion` constant, either directly (`TransportVersions.V_8_2_0`) or indirectly (as it was, with a `TransportVersion` constant on `VersionCompatibilityChecks`)

Reply:
I'd have preferred to keep just one "source of truth", since for that version the mapping should be safe, I think, but I've updated it to how it was before.